charts/fission-all/values.yaml

```diff
@@ -457,6 +457,7 @@ serviceMonitor:
   ##namespace in which you want to deploy servicemonitor
   ##
   namespace: ""
+  additionalServiceMonitorLabels: {}
```

Comment:
Please add description for `additionalServiceMonitorLabels` 
- Whats the purpose?
- How to use it?

Refer rest of values file for reference.

Reply:
Added comment and usage example for the value